admin.js

```diff
@@ -109,20 +109,23 @@
     helpHook = function (moduleName, origionalHelp, config) {
         return function(commandPrefix, event) {
             if (getHasPermission(config, event.sender_id, event.sender_name, event.thread_id, moduleName)) {
-               return origionalHelp.call(this, commandPrefix);
+                return origionalHelp.call(exports.platform, commandPrefix);
             }
             return false;
         };
     };
 
 exports.load = function () {
```

Comment:
you do know that as per the docs, there is an argument passed to this function (hint: its `platform`) and `exports.platform` is not garenteed to be set when it is called